package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netlify/build-image",
-  "version": "3.4.4",
+  "version": "3.6.1",
```

Comment:
What are your thoughts on removing `package-lock.json` from `.gitignore` (separately)?

Reply:
Yeah, I guess we could do it 👍